src/features/home/components/RecommendationModule.tsx

```diff
@@ -28,6 +29,8 @@ type RecommendationModuleProps = {
 export const RecommendationModule = (props: RecommendationModuleProps) => {
   const { display, isBeneficiary, position, index, setRecommendationY, hits } = props
 
+  const hitsData = useHomeOffersModuleData(hits, display.layout)
```

Comment:
Proposition de naming : 
```suggestion
  const selectedHits = useSelectHitsAccordingToLayout(hits, display.layout)
```

Reply:
selectedXXXX veut dire pour moi que l'utilisateur a selectionnés lui meme les choses. D'autant plus que la prop initial du composant elle meme s'appelle juste "hit", donc j'ai simplifié en "hitsData" comme si j'allais l'utiliser dans une flatlist dont la prop "data" est utilisé pour récupérer les données.

Reply:
ce qui me cause soucis avec cela c'est que le type de l'input et de l'output sont les mêmes...

Reply:
en gros tes fonctions mélange `hits`, `homeOffersModuleData` et `hitsData`.
Pourtant on a qu'un seul type de donnée. 
Je te laisse faire une contre-proposition.

Reply:
Je plussois avec Ben, le return doit apparaitre, je propose un compris : `const selectedHits = useLayoutHits(display.layout, hits)` :D

Reply:
Je suis d'accords :) je change